pkg/controller/perconaservermongodb/mgo.go

```diff
@@ -203,6 +209,13 @@ func (r *ReconcilePerconaServerMongoDB) reconcileCluster(ctx context.Context, cr
 
 	}
 
+	if replset.Arbiter.Enabled && !cr.Spec.Sharding.Enabled {
+		err := mongo.SetDefaultRWConcern(ctx, cli, mongo.DefaultReadConcern, mongo.DefaultWriteConcern)
```

Comment:
This `mongo.SetDefaultRWConcern` operation can happen more than once, here and on line 184, I guess it is safe, but just wanted to make sure.